pandas/tests/plotting/test_hist_method.py

```diff
@@ -1,12 +1,12 @@
 """ Test cases for .hist method """
 
-import numpy as np
-from numpy.random import randn
+from numpy import int64
+from numpy.random import choice, normal, rand, randint, randn
```

Comment:
I do like such changes, but in the future, it would be nicer to have such cleanup change as follow-on PR so that it won't distract people from reviewing the "most important" code change ^^